RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -2146,4 +2146,10 @@ Would you like to import them to Rubberduck?</value>
   <data name="CodeMetrics_Nesting" xml:space="preserve">
     <value>Maximum Nesting</value>
   </data>
+  <data name="GeneralSettings_HotkeyRestrictionLabel" xml:space="preserve">
+    <value>Hotkeys should include either Alt and/or Ctrl.</value>
```

Comment:
I'd have used something like: "Hotkeys must include one of Alt or Ctrl.", which is more explicit and not a suggestion...